client/utils/needs-verify.js

```diff
@@ -9,15 +9,15 @@ const needsVerify = (method, user, settings) => {
 
   if (method === "mobile_phone") {
     return (
-      user.method === "mobile_phone" &&
+      (user.method === "mobile_phone" || user.radius_user_token !== "") &&
       user.is_verified === false &&
       settings.mobile_phone_verification
     );
   }
 
   if (method === "bank_card") {
     return Boolean(
-      user.method === "bank_card" &&
+      (user.method === "bank_card" || user.radius_user_token !== "") &&
```

Comment:
also here and above I wonder why is this needed?

Reply:
Routing and redirection is based on `needsVerify` function that's why I have to change this to take effect in the organization-wrapper component.